base/applications/osk/main.c

```diff
@@ -116,78 +117,262 @@ DWORD WINAPI OSK_WarningDlgThread(LPVOID lpParameter)
  */
 VOID OSK_About(VOID)
 {
-    WCHAR szTitle[MAX_BUFF];
-    WCHAR szAuthors[MAX_BUFF];
+    WCHAR szAuthors[MAX_PATH];
     HICON OSKIcon;
 
     /* Load the icon */
     OSKIcon = LoadImageW(Globals.hInstance, MAKEINTRESOURCEW(IDI_OSK), IMAGE_ICON, 0, 0, LR_DEFAULTSIZE);
 
     /* Load the strings into the "About" dialog */
-    LoadStringW(Globals.hInstance, STRING_OSK, szTitle, countof(szTitle));
-    LoadStringW(Globals.hInstance, STRING_AUTHORS, szAuthors, countof(szAuthors));
+    LoadStringW(Globals.hInstance, STRING_AUTHORS, szAuthors, _countof(szAuthors));
 
     /* Finally, execute the "About" dialog by using the Shell routine */
-    ShellAboutW(Globals.hMainWnd, szTitle, szAuthors, OSKIcon);
+    ShellAboutW(NULL, Globals.szTitle, szAuthors, OSKIcon);
```

Comment:
If you set the parent window to NULL, the "About" dialog box will be an orphaned window dialog. Is that what you really want?

Reply:
It was crashing with the window handle set as the parent